homeassistant/components/cover/mqtt.py

```diff
@@ -58,6 +81,8 @@ def async_setup_platform(hass, config, async_add_devices, discovery_info=None):
         config.get(CONF_NAME),
         config.get(CONF_STATE_TOPIC),
         config.get(CONF_COMMAND_TOPIC),
+        config.get(CONF_TILT_COMMAND_TOPIC),
+        config.get(CONF_TILT_STATUS_TOPIC),
```

Comment:
You want to add it to the MQTT cover schema [here](https://github.com/home-assistant/home-assistant/pull/7347/files#diff-2ae9aeb883a17b0e6a77441d5bf68093R58). It will just be new lines that look similar to the ones I linked above. Let me know if you have more questions.